### ISSUE TITLE
Lint doesn't work when invalid tags in html

### ISSUE THREAD

**Issue by ajithr** [htmllint/htmllint#195]
When the html page have invalid tags, htmllint doesn't work. If I have a invalid br tag in the html, then the htmllint doesn't throw any lint error on all html pages. 

**Replocation Procedure:**

1. Set some lint errors in more than one html pages.
2. Use `<br />` in a html page and all html pages reported as passed.
3. Change `<br />` to `</br>`, now lint error throws in all pages.

**Comment by mlochbaum** [comment]
This sounds like a crash, but I don't see one when using `<br />` in the current htmllint. What version of htmllint are you using, and how are you using it?

**Comment by danielwagner** [comment]
I have the same problem. As soon as htmllint encounters a file with a `<br/>` tag, it exits silently. I'm using v0.6.0 by way of gulp-html-lint.
I tried adding a catch block to the promise returned by `htmllint(file.contents.toString(), htmlLintOptions)` in lib/index.js:49 and logging the error:

```
{ [Error: Could not lint html file: foo.html]
  message: 'Could not lint html file: foo.html',
  showStack: false,
  showProperties: true,
  plugin: 'gulp-html-lint',
  __safety: { toString: [Function: bound ] } }
```

**Comment by yvanavermaet** [comment]
@ajithr & @danielwagner Are you sure it's related to the br-element?

I can't reproduce the issue (on a mac). My code:
```javascript
#! /usr/bin/env node
const htmllint = require('htmllint')

const s1 = `<!DOCTYPE html>
<html>
<head>
    <title>Test</title>
</head>
<body>

<span id="my_id" id="my_other_id">qsmhs</span>
<br />
<span id="yet_another_id" id="too_many_ids">qsmhs</span>

</body>
</html>
`;

async function test() {
  console.log(await htmllint(s1, {}));
}

test();
```

Store the above in a .js file (eg: index.js), open a command prompt and execute "node index.js". I get 2 errors as expected.

**Comment by mlochbaum** [comment]
Still can't reproduce this, but the error handling in v0.7.0 is much improved in general, so maybe it's worth checking if you're having problems with `<br />`.
